.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.6.5
+current_version = 4.0.0
```

Comment:
as commented on slack, we should probably do 3.7.0

Reply:
I suspect it might run into issues bumping Flask as a major version, without also bumping `microcosm-flask` as a major version.

Mostly concerned upgrades from `3.X` will probably run into a conflict with the `Flask<2` constraint in other `microcosm-*` and `marquez` libraries

Reply:
so what's next action from here?